modules/consentManagement.js

```diff
@@ -33,33 +32,63 @@ const cmpCallMap = {
 };
 
 /**
- * This function handles interacting with the AppNexus CMP to obtain the consentString value of the user.
+ * This function handles interacting with the AppNexus CMP to obtain the consentObject value of the user.
  * Given the async nature of the CMP's API, we pass in acting success/error callback functions to exit this function
  * based on the appropriate result.
- * @param {function(string)} cmpSuccess acts as a success callback when CMP returns a value; pass along consentString (string) from CMP
+ * @param {function(string)} cmpSuccess acts as a success callback when CMP returns a value; pass along consentObject (string) from CMP
  * @param {function(string)} cmpError acts as an error callback while interacting with CMP; pass along an error message (string)
  */
 function lookupIabConsent(cmpSuccess, cmpError) {
-  let callId = 0;
-  let getConsentDataReq = {
-    __cmp: {
-      callId: 'iframe:' + (++callId),
-      command: 'getConsentData'
-    }
-  };
+  // let callId = 0;
 
   if (window.__cmp) {
-    window.__cmp('getConsentData', 'vendorConsents', cmpSuccess);
+    window.__cmp('getVendorConsents', null, cmpSuccess);
```

Comment:
if CMP js library is not yet loaded then __cmp will still be an object, not a function, good to have a isFunction check.